src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,37 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <a className="UserInfo" href={post.user?.email}>
```

Comment:
you could move `a` tag to `UserInfo` as well

Reply:
not fixed 
<img width="622" alt="image" src="https://user-images.githubusercontent.com/102411130/231686482-2598abd6-b4cd-40d9-8372-e5ac98598a38.png">